lib/sensors/accelerometer.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <unistd.h>
+
+#include <cstdint>
+#include <cstdio>
+#include <optional>
+
+#include "core/types.hpp"
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+#include <io/hardware_i2c.hpp>
+
+namespace hyped::sensors {
+
+static constexpr std::uint8_t kCTRL1Addr        = 0x20;
+static constexpr std::uint8_t kCTRL1Value       = 0x64;
```

Comment:
what is CTRL?

Reply:
name of register, @Platiniom64 reckon its worth mentioning that the names are taken directly from the datasheet